src/ResourceManager/DataLakeStore/ChangeLog.md

```diff
@@ -19,6 +19,7 @@
 -->
 ## Current Release
 * For `Import-AzureRMDataLakeStoreItem` and `Export-AzureRMDataLakeStoreItem` trace logging has been disabled by default to improve performance. If trace logging is desired please use the `-DiagnosticLogLevel` and `-DiagnosticLogPath` parameters
+* Fixed a bug that would sometimes cause PowerShell to crash when uploading lots of small file to ADLS.
```

Comment:
@cormacpayne yes, and it was fixed in the SDK and an update was needed to the SDK package here. There was a null reference exception that was not being properly handled in the SDK that was getting thrown all the way out and ultimately causing PowerShell process to crash.